write-fonts/src/tables/gvar.rs

```diff
@@ -325,22 +325,41 @@ impl GlyphDelta {
     }
 }
 
+pub struct AxisCoordinates {
```

Comment:
if this is public api we'll want docs, and we'll also need some way to construct it.